src/main.js

```diff
@@ -398,7 +409,7 @@ export async function main() {
             describe: 'TCP port to listen on, choose something unique',
           })
           .coerce('addr', addr => {
-            if (!/^[\d\.]+$/.test(addr)) {
+            if (!/^[\d.]+$/.test(addr)) {
```

Comment:
oof, good catch!

Reply:
hrm, ok, actually both are equivalent (I misread the delta as backwards and thought I'd written `[\d.]`, and thought that would match anything, and that would have been an "oof"). I went and tested it and `.` in a character class behaves differently than outside one (inside, it acts as a literal period, outside it matches everything). I usually use `\.` in both contexts to make it clear, but apparently eslint is configured to reject unnecessary escapes. I'd like to stick with the more-visible one, though, is there a config knob to allow `[\.]` but maybe still complain about other unnecessary escapes?

Reply:
It looks like there are no options for this rule, but that we can turn it off. At this point, I'm inclined to leave it on and disable it intentionally and selectively line by line unless it happens so often that it becomes a problem. What do you think?